orchestration/src/test/java/com/sap/ai/sdk/orchestration/OrchestrationUnitTest.java

```diff
@@ -393,4 +393,54 @@ void testEmptyChoicesResponse() {
 
     assertThat(result.getContent()).isEmpty();
   }
+
+  @Test
+  void testExecuteRequestFromJson() {
+    stubFor(post(anyUrl()).willReturn(okJson("{}")));
+
+    prompt = new OrchestrationPrompt(Map.of());
```

Comment:
Please have a look at the following json, that I copied from the Launchpad.

```json
{
    "module_configurations": {
        "llm_module_config": {
            "model_name": "mistralai--mistral-large-instruct",
            "model_params": {},
            "model_version": "2407"
        },
        "templating_module_config": {
            "template": [
                {
                    "role": "user",
                    "content": "How are you?"
                }
            ],
            "defaults": {}
        }
    }
}
```

I would assume, I've got everything from the JSON and should be able to run it with a function like
```
myFunction(json);
```
The second param is only needed (so it should be optional), when I want to pass a `inputParam` or providing message history like the following screenshot:

<img width="1268" alt="Screenshot 2024-11-21 at 20 55 47" src="https://github.com/user-attachments/assets/ad9076c8-dede-4707-9d1e-86ac47be40a8">

Reply:
Ah true, Launchpad doesn't force you to use any variables. Okay, will update 👍🏻

Reply:
After thinking about this again, I actually think the use case for this is not very strong. Because if there are no input params or messages history, prompt execution will always yield the same or very similar results (depending on the temperature setting).

The same is true when using template references. If we really want to support such usage, we could consider allowing `new OrchestrationPrompt()` for these cases. But making the `prompt` itself optional would mean we add 2-3 method overloads for non-productive cases..